config/initializers/react_on_rails.rb

```diff
@@ -48,7 +52,10 @@
   # Default is true. Logs server rendering messages to Rails.logger.info
   config.logging_on_server = true
 
-  config.raise_on_prerender_error = false # change to true to raise exception on server if the JS code throws
+  # Change to true to raise exception on server if the JS code throws. Let's do this only if not
+  # in production, as the JS code might still work on the client and we don't want to blow up the
+  # whole Rails page.
+  config.raise_on_prerender_error = !Rails.env.production?
```

Comment:
It makes a lot of sense to raise from server rendering, except when in production.